packages/trader/src/Modules/Trading/Containers/trade-params-mobile.jsx

```diff
@@ -20,135 +20,133 @@ const DEFAULT_DURATION = Object.freeze({
 const makeGetDefaultDuration = (trade_duration, trade_duration_unit) => duration_unit =>
     trade_duration_unit === duration_unit ? trade_duration : DEFAULT_DURATION[duration_unit];
 
-class TradeParamsModal extends React.Component {
-    constructor(props) {
-        super(props);
-        const { amount, duration, duration_unit } = this.props;
-        const getDefaultDuration = makeGetDefaultDuration(duration, duration_unit);
+const TradeParamsModal = props => {
+    const { amount, duration, duration_unit } = props;
 
```

Comment:
we can define all props here and remove all "props." from component 
```suggestion
const TradeParamsModal =  ({ amount, duration, duration_unit, disableApp, enableApp ,form_components, is_open, toggleModal }) => {
```